snet_cli/mpe_client_command.py

```diff
@@ -149,14 +157,10 @@ def call_server_lowlevel(self):
 
     # III. Stateless client related functions
     def _get_channel_state_from_server(self, grpc_channel, channel_id):
-        # Compile protobuf if needed
-        codegen_dir = Path.home().joinpath(".snet", "mpe_client", "state_service")
-        proto_dir   = Path(__file__).absolute().parent.joinpath("resources", "proto")
-        if (not codegen_dir.joinpath("state_service_pb2.py").is_file()):
-            compile_proto(proto_dir, codegen_dir, proto_file = "state_service.proto")
-
-        # make PaymentChannelStateService.GetChannelState call to the daemon
-        stub_class, request_class, _ = import_protobuf_from_dir(codegen_dir, "GetChannelState")
+        # We should simply statically import everything, but it doesn't work because of the following issue in protobuf: https://github.com/protocolbuffers/protobuf/issues/1491
```

Comment:
Is it worth to add `resources/proto` to `sys.path`?
```
proto_dir = Path(__file__).absolute().parent.joinpath("resources", "proto")
if str(proto_dir) not in sys.path:
    sys.path.append(str(proto_dir))

from state_service_pb2 import ChannelStateRequest as request_class
from state_service_pb2_grpc import PaymentChannelStateServiceStub as stub_class
```
But I'm not sure that this way would be better than yours (only if this can be re-use in multiple parts)

Reply:
Let's consider it later when this story with monorepo will end.